packages/server/routes/api/auth.js

```diff
@@ -107,4 +114,89 @@ router.get("/google/callback", passport.authenticate("google", {
   response.redirect(`icebreak://login?token=${newToken}`);
 });
 
-module.exports = router;
\ No newline at end of file
+router.post("/register", async (request, response) => {
+
+  try{
+    const { email, password, } = request.body;
+    
+    let emailCheck = /@/;
+    let whitespaceCheck = /\s/;
```

Comment:
The regex here is valid, but not strong enough. I recommend using something like this:
https://regexr.com/3e48o

If your regex is strong enough there's no need to test for an `@` and whitespace separately.